.github/workflows/checks.yml

```diff
@@ -41,6 +41,8 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v5
+    - name: Remove Unnecessary Files
+      uses: ./.github/actions/cleanup
```

Comment:
The `lint` job started consistently failing with the following error ([example](https://github.com/orca-so/whirlpools/actions/runs/18689345955)):

```bash
    = note: /usr/bin/ld: final link failed: No space left on device
            collect2: error: ld returned 1 exit status
```

This change alleviates that isssue.